src/v2/repositories/implementations/xcm/MoonbeamXcmRepository.ts

```diff
@@ -16,12 +16,18 @@ import { Symbols } from 'src/v2/symbols';
 import Web3 from 'web3';
 import { TransactionConfig } from 'web3-eth';
 import { AbiItem } from 'web3-utils';
-import { capitalize } from './../../../../hooks/helper/common';
+import { capitalize } from 'src/hooks/helper/common';
 import { isValidEvmAddress } from 'src/config/web3';
 import { ethers } from 'ethers';
+import { AstarNativeToken } from 'src/v2/config/xcm/XcmRepositoryConfiguration';
 
 type ChainName = 'Moonriver' | 'Moonbeam';
 
+export const MOONBEAM_ASTAR_TOKEN_ID: AstarNativeToken = {
```

Comment:
Removing `astarTokens `from constructor will break  `isAstarNativeToken `method which is used in the repo

Reply:
I have kept `astarTokens` [here](https://github.com/AstarNetwork/astar-apps/blob/729ac4047bdbc29b7489463cf0d47d2e5a4e5743/src/v2/repositories/implementations/xcm/MoonbeamXcmRepository.ts#L48)

Reply:
I am sorry. I overlooked that